src/vtk/PolygonWidget/behavior.ts

```diff
@@ -19,6 +21,25 @@ export default function widgetBehavior(publicAPI: any, model: any) {
   model.classHierarchy.push('vtkPolygonWidgetProp');
   model._isDragging = false;
 
+  // If creating new handle and immediately dragging,
+  // widgetManager.getSelections() still points to the last actor
+  // after the mouse button is released. In this widgets case the last actor is part of the LineGlyphRepresentation.
+  // So overUnselectedHandle tracks if the mouse is over a handle so we don't create a new handle
+  // when clicking on segment without immediate mouse move.
+  // A mouse move event sets overUnselectedHandle to false as we can then rely on getSelections().
+  let overUnselectedHandle = false;
+
+  // Check if mouse is over segment between handles
+  const checkOverSegment = () => {
+    // overSegment guards against clicking anywhere in view or over seasoned handle.
```

Comment:
What is a "seasoned" handle?

Reply:
I'll update comment to this:
```
    // overSegment guards against clicking anywhere in view or
    // if over handle that was created before a mouse move event.
```

Also, you can hold off on the testing.  Behavior.ts is horrably broken right now.